packages/pigeon/test/pigeon_test.dart

```diff
@@ -0,0 +1,18 @@
+import 'package:pigeon/pigeon.dart';
+import 'package:test/test.dart';
+
+void main() {
+  test('Should be able to import JavaOptions', () async {
+    const JavaOptions javaOptions = JavaOptions();
+    expect(javaOptions, isNotNull);
+  });
+  test('Should be able to import ObjcOptions', () async {
```

Comment:
Please be consistent about adding a blank line between tests.

Reply:
Added a blank space